gammapy/modeling/models/time.py

```diff
@@ -123,6 +125,74 @@ def evaluate_norm_at_phase(self, phase):
         fp = self.table["NORM"]
         return np.interp(x=phase, xp=xp, fp=fp, period=1)
 
+    #    @property
+    def ontime(self, t_min, t_max):
+        """On time (`~astropy.units.Quantity`)"""
+
+        t_min = Time(t_min)
+        t_max = Time(t_max)
+
+        ontime = t_max - t_min
+        return u.Quantity(ontime.sec, "s")
+
+    def _get_time_meta(self, t_min, t_max):
```

Comment:
As far as I can tell this method is unused. Please remove.

Reply:
Done